backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/StubRenderer.kt

```diff
@@ -18,6 +22,10 @@ object StubRenderer {
                 }
                 +"*/"
             }
+
+            // Let's try to keep non-trivial kdoc formatting intact
+            this.descriptor?.extractKDocString()?.let { +it }
```

Comment:
Sorry, one more concern:
when the flag is not enabled, this would work differently for "two-stage" and "single-stage" compilation. The latter is generally not used in Gradle projects, but the inconsistency is still not entirely ok, as well as enabling the "experimental" feature by default in single-stage mode.